net-dns/dnsdist/dnsdist-1.2.0.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Both vulnerabilities are fixed in this version, correct? So why not just remove the old version instead?